src/gz21_ocean_momentum/step/data/coarsen.py

```diff
@@ -33,6 +108,8 @@ def advections(u_v_field: xr.Dataset, grid_data: xr.Dataset):
         "xu_ocean": u_v_field.coords["xu_ocean"],
         "yu_ocean": u_v_field.coords["yu_ocean"],
     }
+    # TODO got "ValueError: zero-size array to reduction operation fmin which has
```

Comment:
Check this

Reply:
Resolved this: added validation for bounding boxes (which is a prime reason this operation would fail in that way).